docs/networking/best-practice-non-vlan-aware-switch.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 51
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Network
+  - network
+  - VLAN
+  - vlan
+Description: Harvester is built on top of Kubernetes, and uses the [CNI](https://github.com/containernetworking/cni) as the interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, the Harvester UI integrates the network configuration in order to provide a user-friendly way to configure networks for VMs.
+---
+
+# Best Practice for Non VLAN-aware Switch
+
+In this best practice guide for "non VLAN-aware" switch, also known as "dummy" switch, we will introduce Harvester VLAN network and external switch configuration for common scenario.
+
+## Architecture
+
+Hardware:
+- Three Harvester servers with only one single port network card.
+- One or more "non VLAN-aware" switch(es).
+
+Network Specification:
+- The host and the VM networks are in the same subnet.
+
+Cabling:
+- The Harvester servers are connected to the switch in a port from `1` to `3`.
+
+The following diagram illustrates the cabling used for this guide:
+
+   ![non-vlan-aware-case.png](assets/non-vlan-aware-case.png)
+
+## External Switch Configuration
+
+Typically, a "non VLAN-aware" switch cannot be configured.
+
+## Create a VLAN Network in Harvester
+
+You can create a new VLAN network in the **Advanced > Networks** page, and click the **Create** button.
+
+Specify the name and VLAN ID that you want to create for the VLAN network <small>(You can specify the same VLAN ID in different namespaces if you have [Rancher multi-tenancy](/rancher/virtualization-management/#multi-tenancy) configured)</small>.
+   ![create-vlan-network.png](assets/create-network.png)
+
+### Connect a VM to the Harvester management network
+
+The "non VLAN-aware" switch will only send out untagged network traffic to the management network. In Harvester, the untagged traffic is received in VLAN 1.
+
+If you need a VM to connect to the management network, you have to create a VLAN Network in Harvester with VLAN ID 1.
+
+   ![non-vlan-aware-vlan1.png](assets/non-vlan-aware-vlan1.png)
+
+Please refer to [this page](/networking/harvester-network/) for additional information on Harvester Networking.
```

Comment:
the link is broken, please use the [other pages](https://raw.githubusercontent.com/harvester/docs/main/docs/rancher/rke2-cluster.md) as a reference.

Reply:
i used `mkdocs serve` in local, the link works fine.
and i follow it from `harvester-network.md`

Reply:
https://github.com/harvester/docs/blame/main/docs/networking/harvester-network.md#L70